runtime/init.go

```diff
@@ -310,3 +314,14 @@ func (d *Default) decodeJoinCmd(cmd string) {
 	}
 	logger.Debug("joinToken: %v\nTokenCaCertHash: %v\nCertificateKey: %v", d.JoinToken, d.TokenCaCertHash, d.CertificateKey)
 }
+
+// get nodes list,check the config of kubectl
+func (d *Default) getClusterNodes() error {
+	output, err := utils.CmdOutput("kubectl", "get", "nodes")
```

Comment:
If we get nodes immediately, the state of the node is likely to be NotReady.

Reply:
I think this is just a staged verification.I can verify whether the kubectl‘s configuration is correct and whether the master starts normally. This can detect problems in advance.

Reply:
I think we can take all the psychic in an individual command, like sealer check --post(--pre). If show the Not Ready status, user may be confused about it.